controllers/front/CartController.php

```diff
@@ -504,11 +504,16 @@ private function shouldProductsAvailable()
     {
         $product = $this->context->cart->checkQuantities(true);
 
-        return ($product !== true)
-            ? $this->trans(
-                'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted',
-                array('%product%' => is_array($product) ? $product['name'] : ''),
-                'Shop.Notifications.Error')
-            : true;
+        if ($product !== true && is_array($product)) {
+            $errorMsg = $product['active']
+                ? 'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.'
+                : 'This product (%product%) is no longer available.';
+            return $this->trans(
+                $errorMsg,
+                array('%product%' => $product['name']),
+                'Shop.Notifications.Error');
+        }
```

Comment:
@marionf as you can see right here we put the message in translate function, so you just need to translate this field.
Can you check this with @LouiseBonnard ?

Thanks ;)